project/ScalafixBuild.scala

```diff
@@ -194,15 +194,27 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
         "unit/test" ::
         "docs/run" ::
         "interfaces/doc" ::
+        s"++$scala213V1" ::
```

Comment:
instead of hardcoding the previous versions, you can probably generate the list dynamically and loop over it

Reply:
I have more tests that fails when testing against more scala Version. 
I was thinking about adding a specific test that will test only ExplicitResultType : 
`+unit/testOnly scalafix.tests.rule.RuleSuite -- -z explicitResult `